schema/artwork/index.js

```diff
@@ -251,25 +251,16 @@ export const artworkFields = () => {
     },
     is_buy_nowable: {
       type: GraphQLBoolean,
-      description: "When in an auction, can the work be bought before any bids are placed",
+      description: "When in an auction, can the work be bought immediately",
       resolve: ({ id, acquireable, sale_ids }, options, request, { rootValue: { salesLoader } }) => {
         if (sale_ids && sale_ids.length > 0 && acquireable) {
           return salesLoader(id, {
             id: sale_ids,
             is_auction: true,
-            auction_state: "open",
+            live: true,
```

Comment:
Just for future developer reference... you can't query the `sales` endpoint with an `auction_state` param (it will not have any effect), the correct query for getting live auctions is `?live=true`.